lib/buildmodel.js

```diff
@@ -135,6 +135,81 @@ const BuildModel = class {
     stream(config, response) {
         return this.executor.stream(config, response);
     }
+
+    /**
+     * Get a build based on id
+     * @param  {String}   id                The key of the record to retrieve
+     * @return {Function} callback
+     */
+    get(id, callback) {
+        const config = {
+            table: 'builds',
+            params: {
+                id
+            }
+        };
+
+        this.datastore.get(config, (err, data) => {
+            if (err) {
+                return callback(err);
+            }
+            if (!data) {
+                return callback(new Error('Record does not exist'));
```

Comment:
Should the model's be the ones responsible for bubbling up Boom errors? If not, how do we plan on knowing that this is a 404 ?